src/novmm/loader/linux_setup.go

```diff
@@ -21,9 +21,11 @@ package loader
 #include <string.h>
 
 #ifdef __x86_64__
-#include <x86_64-linux-gnu/asm/bootparam.h>
+//#include <x86_64-linux-gnu/asm/bootparam.h>
+#include <asm/bootparam.h>
 #else
-#include <i386-linux-gnu/asm/bootparam.h>
+//#include <i386-linux-gnu/asm/bootparam.h>
+#include <asm/bootparam.h>
```

Comment:
Thanks, I got it compiled for Fedora, hoping you can come up solution from upstream for both Fedora and Ubuntu.